### PR TITLE
Upgrade to support Angular 14

### PR DESCRIPTION
**Pull request by abhishek1234321**

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
   "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "cli": {
```
Comment (abhishek1234321) [inline angular.json:3]: I can remove this off, it was just annoying

File: tsconfig.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
     "moduleResolution": "node",
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
-    "target": "es5",
+    "target": "ES2015",
```
Comment (abhishek1234321) [inline tsconfig.json:14]: https://caniuse.com/?search=es6

**Comment by jannicz** [comment]
**SOLVED**

I created an own [PR](https://github.com/jannicz/ng-svg-icon-sprite/pull/79) that also updates some other packages and fixes the docs build for GitHub pages. I am having trouble with an import statement in `app.module.ts` [L7](https://github.com/jannicz/ng-svg-icon-sprite/pull/79/files#diff-502d295b435b10ba99f5377504ca51149312e3cbebbbaa78ac5d925cb04cbfb0R7). 

If I use the shortcut syntax (as if importing an npm package) `import { IconSpriteModule } from 'ng-svg-icon-sprite'` I will get following compilation errors. Do you have an idea why? It occurs since the update to 14.

ngModule error

But even without fixing this, probably the lib will work. It's just the development app that seems to have the problem.

**Comment by jannicz** [comment]
Based on the work of @abhishek1234321 (many many thanks) I was able to upgrade the lib to Angular 14. The version name is 1.12.0 and its already published to npm, see https://www.npmjs.com/package/ng-svg-icon-sprite

Related PR: https://github.com/jannicz/ng-svg-icon-sprite/pull/79

**Comment by abhishek1234321** [comment]
hey, thanks much! I couldn't get to your comments earlier

**Pull request closed without merging**